tests/config-kmdf/Cargo.lock

```diff
@@ -47,7 +47,7 @@ version = "1.1.3"
 source = "registry+https://github.com/rust-lang/crates.io-index"
```

Comment:
these lock diffs are from #492, since nothing updated the tests folders locks before that pr merged